portal-web/test/functional/com/liferay/portalweb/macros/BlogsEntry.macro

```diff
@@ -420,6 +420,28 @@ definition {
 			entryContentEdit = "${entryContentEdit}");
 	}
 
+	macro editEntryInlineImageURL {
+		if (isSet(uploadFileName)) {
+			Click(
+				key_uploadFileName = "${uploadFileName}",
+				locator1 = "BlogsEntry#CONTENT_INLINE_IMAGE");
+
+			Click(locator1 = "AlloyEditor#TEXT_FORMATTING_LINK");
+
+			Type(
+				locator1 = "AlloyEditor#TEXT_FORMATTING_LINK_INPUT",
+				value1 = "${entryInlineImageUrl}");
+
+			Click(locator1 = "AlloyEditor#TEXT_FORMATTING_LINK_INPUT_CONFIRM");
+
+			Click(
+				key_uploadFileName = "${uploadFileName}",
+				locator1 = "BlogsEntry#CONTENT_INLINE_IMAGE");
+
+			AssertVisible(locator1 = "AlloyEditor#IMAGE_TOOLBAR");
+		}
```

Comment:
@Jayda-4145 what I was trying to explain here is that you can take all of this if{} block and keep it as an if{} block, but at the end of (_within_) the `addContentWithInlineImage` macro.